tutorials/flow/helloworld/maven/01__meecrowave/content.adoc

```diff
@@ -0,0 +1,99 @@
+= HelloWorld - with Apache meecrowave - Ramp up in a second.
+
+:title: HelloWorld - with Apache meecrowave - Ramp up in a second.
+:authors: sven ruppert
+:type: text, video
+:topic: helloworld
+:tags: apache, meecrowave, flow, v10
+:description: How to start a Vaadin Flow project with Apache meecrowave in a second
+:repo: https://github.com/vaadin-learning-center/flow-helloworld-maven-meecrowave
+:linkattrs:
+:imagesdir: ./images
+
+
+A nano project to start a Vaadin project. Perfect for Micro-UIs packed as fat jar in a docker image.
+This will be used in some of the tutorial implementations.
+
+== target of this project
+
+The target of this project is a minimal rampup time for a first hello world.
+Why we need one more HelloWorld? Well, the answer is quite easy.
+If you have to try something out, or you want to make a small POC to present something,
+there is no time and budget to create a demo project.
+You don´t want to copy paste all small things together.
+Here you will get a HelloWorld-Project that will give you all in a second.
+
+Clone the repo and start editing the class `VaadinApp` or `BasicTestUIRunner`.
+Nothing more. 
+
+== How does it work?
+
+This project will not use any additional maven plugin or technology.
+
+Here we are using the plain *meecrowave* as Servlet-Container.
+http://openwebbeans.apache.org/meecrowave/index.html[http://openwebbeans.apache.org/meecrowave/index.html]
+
+But let´s start from the beginning.
+
+== Start the Servlet-Container (Java)
+
+The class `BasicTestUIRunner` will ramp up the Container.
+
+Here all the basic stuff is done. The start will init. a ServletContainer at port *8080*.
+If you want to use a random port, use `randomHttpPort()` instead of `setHttpPort(8080);`
+The WebApp will deployed as *ROOT.war*. 
+
+[source,java]
+----
+public class BasicTestUIRunner {
+  private BasicTestUIRunner() {
+  }
+
+  public static void main(String[] args) {
+    new Meecrowave(new Meecrowave.Builder() {
+      {
+//        randomHttpPort();
+        setHttpPort(8080);
+        setTomcatScanning(true);
+        setTomcatAutoSetup(false);
+        setHttp2(true);
+      }
+    })
+        .bake()
+        .await();
+  }
+}
+----
+
+After this you can start the app invoking the main-method.
+
+== The UI itself
+
+The UI itself is quite easy. There is only a button you can click. For every click, the counter will be increased.
```

Comment:
Again, refrain from saying that things are easy or simple. Instead, say something like "The UI consists of a button that increases a counter variable when clicked"